### PR TITLE
Problem: REST API does not accept null for string values

### PR DESCRIPTION
**Pull request by dkliban**
Solution: update serializers to accept null

re: #4676
https://pulp.plan.io/issues/4676

### CODE REVIEW

**Comment by bmbouter** [comment]
Are there any alternatives to this implementation? I'm concerned from the [DRF docs](https://www.django-rest-framework.org/api-guide/fields/#charfield) that say "doing so means that there will be two differing types of empty value permissible for string representations, which can lead to data inconsistencies and subtle application bugs."

**Comment by dkliban** [comment]
The alternative is to not accept empty strings and only accept null.

**Comment by bmbouter** [comment]
Adding WIP since we have a meeting to discuss tomorrow w/ @jlsherrill

**Comment by dkliban** [comment]
@bmbouter This is  ready for re-review.

**Comment by bmbouter** [comment]
@dkliban actually it needs a breaking change 3.0 release note...

File: docs/release-notes/pulpcore/3.0.x.rst [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
 
+3.0.0rc3
+========
+
+`Comprehensive list of changes and bugfixes for rc 3 <https://github.com/pulp/pulpcore/compare/3.0.0rc2...3.0.0rc3>`_.
+
+Breaking Changes
+----------------
+
+All the string fields in the REST API no longer accept an empty string as a value. These fields
```
Comment (bmbouter) [inline docs/release-notes/pulpcore/3.0.x.rst]: this looks great, thanks. +1 to merge

**Pull request merged**